brvs-core/src/main/java/iroha/validation/listener/BrvsIrohaChainListener.java

```diff
@@ -139,7 +140,16 @@ public BrvsIrohaChainListener(
 
     if (queryResponse.hasErrorResponse()) {
       ErrorResponse errorResponse = queryResponse.getErrorResponse();
-      throw new ErrorResponseException(errorResponse);
+      final ErrorResponseException responseException = new ErrorResponseException(errorResponse);
+      if (errorResponse.getErrorCode() == 3) {
```

Comment:
Error code could be extracted as a constant

Reply:
Thanks, fixed